cms/static/coffee/spec/main.coffee

```diff
@@ -180,10 +178,13 @@ requirejs.config({
             exports: "sinon"
         },
         "jasmine-imagediff": {},
-        "jasmine-stealth": {
+        "common/js/spec_helpers/jasmine-extensions": {
```

Comment:
I don't like having vendor libraries under ``spec_helpers`` as that's a directory of our own code. Can it just live in the regular vendor directory? Ideally it would be installed via npm, but this one doesn't seem to be available.